src/garage/sampler/is_sampler.py

```diff
@@ -1,23 +1,25 @@
+"""ISSampler."""
 import copy
 from math import exp
 from math import log
 import random
 
 from numpy import var
 
-from garage.sampler import BatchSampler
+from garage.sampler import Sampler
 from garage.sampler.utils import truncate_paths
 
 
-class ISSampler(BatchSampler):
-    """
+class ISSampler(Sampler):
+    """ISSampler.
+
     Sampler which alternates between live sampling iterations using
     BatchSampler and importance sampling iterations.
 
     Args:
         algo (garage.np.algos.RLAlgorithm): An algorithm instance.
         env (garage.envs.GarageEnv): An environement instance.
-        n_backtrack (str/int): Number of past policies to update from
+        n_backtrack (str): Number of past policies to update from
```

Comment:
i'm confused by this arg. `n_backtrack` makes it sound like a number -- how is it a string? The argument comment should at least explain this.